### PR TITLE
E2e scheduled triggering

### PR DESCRIPTION
**Pull request by tnsetting**
# Hey, I just made a Pull Request!

## Description
End to End test for scheduled triggering for Flyte workflow

## Motivation and Context
End to End test for scheduled triggering for Flyte workflow

## Checklist for PR author(s)

- [ ] Changes are covered by unit test
- [ ] All tests pass
- [ ] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [ ] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.e2e_tests;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
```
Comment (honnix) [inline styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java:26]: Use `import static org.hamcrest.MatcherAssert.assertThat;` instead.

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/*-
+ * -\-\-
+ * Spotify End-to-End Integration Tests
+ * --
+ * Copyright (C) 2016 - 2019 Spotify AB
```
Comment (honnix) [inline styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java:5]:
```suggestion
 * Copyright (C) 2016 - 2020 Spotify AB
```

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.e2e_tests;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
```
Comment (honnix) [inline styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringFlyteIT.java:26]: I guess this is gonna fail. It should appear before `import static org.hamcrest.Matchers.is;`

**Review by honnix — APPROVED** [review]

File: styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringIT.java [hunk 4]
```diff
@@ -38,6 +38,21 @@
...
+  @Test
+  public void testScheduledTriggeringFlyteWorkflow() throws Exception {
+    // Generate workflow configuration
+    var workflowJson =
+        Json.OBJECT_MAPPER.writeValueAsString(
+            Map.of(
+                "id", workflowId1,
+                "schedule", "* * * * *",
+                "flyte_exec_conf", FLYTE_EXEC_CONF_MAP));
+    var workflowJsonFile = temporaryFolder.newFile().toPath();
```
Comment (honnix) [inline styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/ScheduledTriggeringIT.java:50]: The file writing part can be put into the shared method as well.

**Review by honnix — APPROVED** [review]

**Comment by honnix** [comment]
:shipit: I will rebase on master later for #892 to keep this PR simple.

**Comment by tnsetting** [comment]
> :shipit: I will rebase on master later for #892 to keep this PR simple.

I will wait until the build finishes.

**Pull request merged**